pkg/config/app/kuma-cp/config.go

```diff
@@ -296,6 +296,7 @@ var DefaultConfig = func() Config {
 		},
 		MeshService: MeshServiceConfig{
 			GenerationInterval: config_types.Duration{Duration: 2 * time.Second},
+			GracePeriod:        config_types.Duration{Duration: 10 * time.Second},
```

Comment:
I'd call it `DeleteGracePeriod`, but feel free to leave it if you think this is more correct.